docs/checkers/flag-deref.md

```diff
@@ -0,0 +1,13 @@
+Detects immediate dereferencing of `flag` package pointers.
+Suggests using `XxxVar` functions to achieve desired effect.
```

Comment:
Can you add a bit more detailed explanation from issue? 😉

Reply:
PTAL.

Reply:
`Dereferencing returned pointers will lead to hard to find errors where flag values are not updated after flag.Parse().`